packages/bloc/test/bloc_test.dart

```diff
@@ -551,7 +551,7 @@ void main() {
           );
           expect(
             (capturedError as StateError).message,
-            'Cannot add new events after calling close',
+            'Cannot add event after closing',
```

Comment:
This test should not be changed, but a broadcast `StreamController` should be used which would give you the correct error.

Reply:
Will do